src/tz_aix.rs

```diff
@@ -5,22 +5,3 @@ use std::io::{BufRead, BufReader};
 pub(crate) fn get_timezone_inner() -> Result<String, crate::GetTimezoneError> {
     env::var("TZ").map_err(|_| crate::GetTimezoneError::OsError)
```

Comment:
while you're in here, can you also apply the patch to avoid panicking on invalid UTF-8 in the `OsString`?

Reply:
whoops, just read your other comment. I stand corrected!